x230/README.md

```diff
@@ -126,14 +132,14 @@ or ethernet to `sudo apt-get install flashrom`
 
 		   Edge of pi (furthest from you)
 		               (UART)
-		 L           GND TX  RX                           CS
-		 E            |   |   |                           |
+		 L           GND                                  CS
+		 E            |                                   |
 		 F +---------------------------------------------------------------------------------+
 		 T |  x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x  |
 		   |  x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x  |
 		 E +----------------------------------^---^---^---^-------------------------------^--+
-		 D                                    |   |   |   |                               |
-		 G                                   3.3V MOSIMISO|                              GND
+		 D                                    |   |   |   |                              
+		 G                                   3.3V MOSIMISO|                              
```

Comment:
why would you remove the uart pins?

Reply:
Because the are not used and create confusion at first. Just kept the connected pins.